### PR TITLE
Refactor code base

### PR DESCRIPTION
**Pull request by chaitanyaenr**
### Description
This commit:
- Refactors the code base to be more modular by moving functions
  into respective modules to make it lean and reusable.
- Uses black to reformat the code to follow PEP 8 practices.

### Fixes
https://github.com/cloud-bulldozer/kraken/issues/99

### CODE REVIEW

File: CI/scenarios/post_action_etcd_example_py.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 def run(cmd):
     try:
         output = subprocess.Popen(
-            cmd, shell=True, universal_newlines=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT
+            cmd, shell=True, universal_newlines=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT,
```
Comment (paigerube14) [inline CI/scenarios/post_action_etcd_example_py.py:9]: Is this comma supposed to be here? Random commas are at the end of lines in a couple of your files. Haven't seen it cause issues though.

File: kraken/litmus/common_litmus.py [hunk 2]
```diff
@@ -2,6 +2,52 @@
 import logging
 import time
 import sys
+import requests
+import yaml
+import kraken.cerberus.setup as cerberus
+
+
+# Inject litmus scenarios defined in the config
```
Comment (paigerube14) [inline kraken/litmus/common_litmus.py:10]: Very small nit pick, all other scenario types you used "run" as the function name and here you used scenarios. Might be good to keep it consistent across all scenario types to use "run" as the main function

File: kraken/time_actions/common_time_functions.py [hunk 3]
```diff
@@ -76,7 +78,7 @@ def skew_time(scenario):
 # From kubectl/oc command get time output
 def parse_string_date(obj_datetime):
     try:
-        date_line = re.search(r"[a-zA-Z0-9_() .]*\w{3} \w{3} \d{2} \d{2}:\d{2}:\d{2} \w{3} " r"\d{4}\W*", obj_datetime)
+        date_line = re.search(r"[a-zA-Z0-9_() .]*\w{3} \w{3} \d{2} \d{2}:\d{2}:\d{2} \w{3} " r"\d{4}\W*", obj_datetime,)
```
Comment (paigerube14) [inline kraken/time_actions/common_time_functions.py:81]: Still a couple more commas in places 👎

File: kraken/litmus/common_litmus.py (hunk 2 above)
Comment (chaitanyaenr) [reply kraken/litmus/common_litmus.py:10]: Absolutely, fixed it.

File: kraken/time_actions/common_time_functions.py (hunk 3 above)
Comment (chaitanyaenr) [reply kraken/time_actions/common_time_functions.py]: Fixed it, thanks.

File: CI/scenarios/post_action_etcd_example_py.py (hunk 1 above)
Comment (chaitanyaenr) [reply CI/scenarios/post_action_etcd_example_py.py]: Fixed it, thanks.

File: kraken/time_actions/common_time_functions.py [hunk 4]
```diff
@@ -134,3 +136,17 @@ def check_date_time(object_type, names):
             if counter < max_retries:
                 logging.info("Date in pod " + str(pod_name[0]) + " reset properly")
     return not_reset
+
+
+def run_scenarios(scenarios_list, config, wait_duration):
```
Comment (paigerube14) [inline kraken/time_actions/common_time_functions.py:141]: sorry one more inconsistent name of the main function.

**Comment by paigerube14** [comment]
LGTM

File: kraken/time_actions/common_time_functions.py (hunk 4 above)
Comment (chaitanyaenr) [reply kraken/time_actions/common_time_functions.py]: Fixed it, thanks.

**Pull request merged**